### ISSUE TITLE
documentation planning for Python and OS versions

### ISSUE THREAD

**Issue by douglasawh** [philosowaffle/peloton-to-garmin#33]
This isn't really _request_, so much as a couple of thoughts about documentation. I wasn't really sure where to put it. 

At some point (I don't know if we are there yet), maybe it makes sense to pull some information out of the README and into separate docs. Some Linux versions, such as Ubuntu, have a new release every six months. I don't think it makes sense to keep the entire list in the README. Linux users are likely to know there way around GitHub to look in docs, and of course there can be a link from README to docs.

Upon my latest pull request, I noticed that it looks like ubuntu, windows, and mac all get automatic tests for python 3.5-3.8. Is that correct? This would be good information for the readme. Probably the ubuntu 3.5-3.8 test is enough. The real issue between Linux distros would be the python version, I would think.

**Comment by philosowaffle** [comment]
Agreed, README is getting lengthy.   Could probably be whittled down to:

* Short description of features
* Quick start instructions for the non-tech savvy
* Disclaimers
* Link to other docs or wiki for more technical instructions

And yes, automated tests have been added. Test coverage is currently abysmal but the framework is now in place.  Seemed there was enough interest in this script that I needed to be more careful about not breaking things.
